foundry/cameraPopoutAppWrapper.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Abstraction of the Application interface to be used with the Draggable class as a substitute for the app
+ * This class will represent one popout feed window and handle its positioning and draggability
+ */
+declare class CameraPopoutAppWrapper {
+
+  /**
+   * The CameraViews application that this popout belongs to
+   */
+  view: CameraViews;
+
+  /**
+   * The div element of this specific popout window
+   */
+  element: JQuery;
+
+  /**
+   * ID of the user this popout belongs to
+   */
+  userId: string;
+
+  /**
+   * @param view- The CameraViews application that this popout belongs to
+   * @param userId- ID of the user this popout belongs to
+   * @param element- The div element of this specific popout window
+   */
+  constructor(view: CameraViews, userId: string, element: JQuery);
```

Comment:
I think the constructor should be above the fields set in it to match the order in foundry.js

Reply:
Fair enough, my habit to put properties first in the class before the constructor, I have moved it to be first.